src/components/Type/Link.module.scss

```diff
@@ -0,0 +1,30 @@
+.Link {
+  color: var(--GrayPrimary--translucent);
+
+  &,
+  &:active,
+  &:hover {
+    text-decoration: none;
+  }
+
+  // Style variants
+
+  &.Standard {
```

Comment:
Should we consider making all these top level now that we're using CSS Modules?

Reply:
Yeah maybe. I went in that direction first and stopped for some reason, lemme think...

Reply:
Yeah, see my other comment...might be able to do this on a later iteration if it's going to hold things up. I'll let you decide but I'm fine with that if you prefer

Reply:
I'd rather keep them nested for now, unless I run into issues with it during QA. I'll update this thread a little later today.